packages/jbrowse-web/src/mst-types.js

```diff
@@ -24,7 +23,6 @@ export const PropTypes = {
 
 export const Region = types
   .model('Region', {
-    assemblyName: types.string,
```

Comment:
you sure we don't want assemblyName in a region anymore? I could see this eventually leading to problems when there are a couple different assemblies in play that have the same chromosome names.

Reply:
I've thought a lot about this, and I still stick with what I did here. I also took assemblyName out of the adapters, since really there is nothing inherent about a data source that defines an assembly. You can say a data source is for a particular assembly, but it's not actually encoded in the file/config/other source. I ran into weird things like trying to use an adapter in an assembly config where defining an assembly name didn't make sense.

Likewise the view shouldn't need to know the assembly, since you might want to display the same Region of several closely related assemblies in the same view.

I think the track needs to manage the assembly. That way you can still have multiple assemblies with the same chromosome names, and each track will just handle them in the way it needs for that assembly. And if you want multiple assemblies in one track, you can use subtracks.

Reply:
what about the case where we want critter:chr1 displayed next to human:chr1 in the same lineargenomeview? The lineargenomeview would not be able to tell the difference between them